app/page.tsx

```diff
@@ -0,0 +1,46 @@
+import { stripeClient } from '../util/stripe';
+import Products from './Products';
+
+export const metadata = { description: 'Products Page' };
+
+export default async function HomePage() {
+  const magazine = await stripeClient.products.retrieve(
+    process.env.MAGAZINE_ID!,
+    {
+      expand: ['default_price'],
+    },
+  );
+
+  const tablet = await stripeClient.products.retrieve(process.env.TABLET_ID!, {
+    expand: ['default_price'],
+  });
+
+  return (
+    <Products
+      tablet={{
+        price:
+          tablet.default_price instanceof Object
+            ? {
```

Comment:
What I meant is to already show an error here (early return further up in this component)